### DISCUSSION TITLE
Lens AI questions off other index/information

### DISCUSSION THREAD

**Discussion by jimsbr** [GitbookIO/community#213] [0. General bugs and questions] [answered]
Want to confirm as well with Lens AI I must manually type CTRL/Command + K and then navigate by pressing Lens to access it?
Is there an easier way to click and access Lens AI for questions?

Also, if a docs page for instance is a platform that interacts with many other applications - can I upload somewhere relevant/or all of another applications documentation so Lens can also answer from that indexed information as well, not just mine?

### What's your desired outcome?

Easily click Lens AI to get an answer on a question, it answers not only based on the info you have but can reference answers available in other relevant docs...

**Comment by addisonschultz** [comment, marked as answer]
Hey @usernameisJim - great suggestions! However, it's unfortunately not possible to add other content to be indexed by Lens right now. We've heard this before, so it's definitely on our radar!
